pyopae/opae/__init__.py

```diff
@@ -0,0 +1,7 @@
+from _opae import properties
+from _opae import token
+from _opae import handle
+from _opae import shared_buffer
+from _opae import event
+from _opae import *
+__all__ = [ 'reconfigure', 'version', 'properties', 'token', 'handle', 'shared_buffer', 'event']
```

Comment:
Do the `fpga_objype`, `fpga_status`, etc. be included here?

Reply:
@buie1 , No, they don't since they are not Python modules. They get included with:
```Python
from _opae import *
```